grid2op/Opponent/RandomLineOpponent.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) 2019-2020, RTE (https://www.rte-france.com)
+# See AUTHORS.txt
+# This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0.
+# If a copy of the Mozilla Public License, version 2.0 was not distributed with this file,
+# you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+# This file is part of Grid2Op, Grid2Op a testbed platform to model sequential decision making in power systems.
+
+import numpy as np
+from grid2op.Opponent import BaseOpponent
+from grid2op.Converter import LineDisconnection
+
+
+class RandomLineOpponent(BaseOpponent):
+    def __init__(self, action_space):
+        # Apply converter
+        converter_action_space = LineDisconnection(action_space)
```

Comment:
I understand now why you coded that. It's interesting, but if you use that class, you need to test it.

Reply:
done